robot/EDA/tests/browser/contacts/verify_preferred_phone.robot

```diff
@@ -162,3 +179,31 @@ Restore eda settings
     Open EDA Settings Tab menu item
     Set the disable preferred phone enforcement
     Shift to default content
+
+Switch to successful
+    Go To Setup Home
+    Wait For New Window                     Home | Salesforce
+    Select window                           Home | Salesforce
+    Wait Until Loading Is Complete
+    Populate Placeholder                    Quick Find          Apex Jobs
+    Open Apex                               Apex Jobs
+    ...                                     Cannot find Apex Jobs page
+    ...                                     false
+
+    Select frame with title                 Apex Jobs ~ Salesforce - Developer Edition
```

Comment:
Should we be concerned that this test seems geared towards Developer orgs only? Do we ever run the robot tests on other types of orgs?

Reply:
So far it's geared towards the test/build environment.